fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayTestTag.kt

```diff
@@ -0,0 +1,27 @@
+package org.mozilla.fenix.tabstray
```

Comment:
nit: license is missing

Reply:
I'm surprised the linter didn't catch this